packages/altair-fastify-plugin/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import { getDistDirectory, renderAltair, RenderOptions } from 'altair-static';
+import fp from 'fastify-plugin';
+import fastifyStatic from 'fastify-static';
+
+import type { FastifyPluginCallback } from 'fastify';
+
+export interface AltairFastifyPluginOptions extends RenderOptions {
+  /**
+   * URL to set as the server endpoint.
+   *
+   * By default is `/graphql`
+   */
+  endpointURL?: string;
```

Comment:
Any reason for having `endpointURL` and `baseUrl` here when they are already in the `RenderOptions` interface?

Reply:
to add the comment `By default is /graphql` to the type definition